include/ccl_core.h

```diff
@@ -333,7 +336,8 @@ ccl_parameters ccl_parameters_create(double Omega_c, double Omega_b, double Omeg
 				     double Neff, double* mnu, int n_mnu,
 				     double w0, double wa, double h, double norm_pk,
 				     double n_s, double bcm_log10Mc, double bcm_etab, double bcm_ks,
-				     double mu_0, double sigma_0, int nz_mgrowth, double *zarr_mgrowth,
+				     double mu_0, double sigma_0, double c1_mg, double c2_mg, double lambda_mg,
+                                     int nz_mgrowth, double *zarr_mgrowth,
```

Comment:
indentation

Reply:
fixed

Reply:
Fixed. There was tabs instead of spaces in some of those lines and I suspect that that what was causing the problem but was not visible on the editor vi.